uiowa/uiowa
# AMCS view updates

Requested changes for the custom faculty view on the AMCS site.

# How to test
`blt ds --site=amcs.uiowa.edu`
`drush @amcs.local uli`
Go to `amcs.uiowa.ddev.site/faculty-information`
Check that it resembles amcs.uiowa.edu/faculty-research, only changes are the URL and using department instead of research areas for the search option and display
(If no departments have been added yet add one to a person to make sure it does display correctly and shows as a search option.)
